extensions/industry/healthcare/fhir/fieldgroups/medicationrequest.schema.json

```diff
@@ -0,0 +1,378 @@
+{
+   "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$id":"https://ns.fhir.org/fieldgroups/medicationrequest",
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Medication",
+   "type":"object",
+   "meta:tags":{
+      "industry":[
+         "healthcare_services"
+      ]
+   },
+   "meta:intendedToExtend":[
+      "https://ns.adobe.com/xdm/classes/medication",
+      "https://ns.adobe.com/xdm/context/profile",
+      "https://ns.adobe.com/xdm/classes/provider"
+   ],
+   "meta:extensible":true,
+   "meta:abstract":true,
+   "description":"An order or request for both supply of the medication and the instructions for administration of the medication to a patient.(Based on HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "medicationRequest":{
+         "properties":{
+            "xdm:healthcareMedicationRequest":{
+               "type":"object",
+               "properties":{
+                  "fhir:identifier":{
```

Comment:
Please change the title from `fhir:identifier` to `xdm:identifier`

Reply:
done!